__tests__/app.test.js

```diff
@@ -380,5 +380,23 @@ describe('GET /api/articles/:article:id/comments', () => {
              expect (articles).toHaveLength(1)
             expect((articles[0].topic)).toBe('cats')
         })
+       });  
+       test('should respond with apropriate error if topic does not exist ', () => {
+        return request(app)
+        .get('/api/articles?topic=dogs')
+         .expect(404)
+         .then((response)=>{
+            expect(response.body.msg).toBe('id not found');
```

Comment:
I don't think that "id not found" is the best error message here. Maybe "topic not found"